apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V53__Tech_debt_db_cleanup.sql

```diff
@@ -0,0 +1,15 @@
+-- both psql and hsql support non-primary key index renaming
```

Comment:
minor nitpick but I think that the name here should reflect what this does a bit better. I propose V53__Remove_unused_db_objects.sql

Reply:
Not above plagarism...used your name! Done